### PR TITLE
Add a `ServerSideRender` block example.

### LINKED ISSUE #129: Add an example that utilizes `ServerSideRender`
The question of how to build a block that utilizes `ServerSideRender` has come up a few times, and the [source documentation](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-server-side-render/) does not provide a complete working example. Therefore, there's an opportunity to add one to this repo.

### PR DESCRIPTION
**Pull request by ndiego**
Fixes #129 

This PR adds a simple block that utilizes `ServerSideRender`.

### CODE REVIEW

**Comment by flexseth** [comment]
Thanks for adding, Nick. I'd love to see an ELI5 for SSR and the benefits/drawbacks/why-to for SSR!

Excellent work as always, sir.

File: plugins/server-side-render-block-d26119/src/edit.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+				>
+					<TextControl
+						label={ __( 'Label', 'block-development-examples' ) }
+						value={ attributes.label }
+						onChange={ ( label ) => setAttributes( { label } ) }
+					/>
+				</PanelBody>
+			</InspectorControls>
+			<div { ...useBlockProps() }>
+				<Disabled>
```
Comment (ryanwelcher) [inline plugins/server-side-render-block-d26119/src/edit.js:26]: Why are we wrapping `ServerSideRender` in the `Disabled` component? Not suggesting it's incorrect, just wondering. It _might_ be worth explaining the reasoning around it in the Readme as well.

**Review by ryanwelcher — APPROVED** [review]
Overall looks great! I had one question that may be a good addition to the Readme but it's not a blocker for me.

File: plugins/server-side-render-block-d26119/src/edit.js (hunk 1 above)
Comment (ndiego) [reply plugins/server-side-render-block-d26119/src/edit.js:26]: Good point, I added a note to the readme.
